Aptoide/aptoide-client-v8
# [ASV-1052] Action Item card with Palette Color

**What does this PR do?**

   Adds the dominant color from the background image of the editorial content to the editorial action item card.

**Database changed?**

 No

**Where should the reviewer start?**

- ImageLoader.java

**How should this be manually tested?**

Go to the EditorialFragment and see if the card that says Game of the Week is with the color of the image (default is light grey) with the current background image is suppose to be dark grey

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-1052

**Questions:**

Should we wait for the right color to display the item or should we show the item right away with the default color and then change the color. The first approach was the approach that was taken


**Code Review Checklist**
